src/main/java/org/folio/dcb/service/impl/CirculationServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package org.folio.dcb.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.dcb.client.feign.CirculationClient;
+import org.folio.dcb.domain.dto.CheckInRequest;
+import org.folio.dcb.domain.entity.TransactionEntity;
+import org.folio.dcb.service.CirculationService;
+import org.springframework.stereotype.Service;
+
+import java.time.OffsetDateTime;
+
+@Service
+@Log4j2
+@RequiredArgsConstructor
+public class CirculationServiceImpl implements CirculationService {
+
+  private final CirculationClient circulationClient;
+
+  @Override
+  public void checkInByBarcode(TransactionEntity dcbTransaction) {
+    log.debug("checkInByBarcode:: automate item checkIn using  dcbTransaction {} ", dcbTransaction);
+    circulationClient.checkInByBarcode(createCheckInRequest(dcbTransaction.getItemBarcode(), "3a40852d-49fd-4df2-a1f9-6e2641a6e91f"));
```

Comment:
@MagzhanArtykov 
I believe the servicePointId shouldn't be just a pure string representation of UUID.
Do you think it could be separated into a constant with a valid description ?

Reply:
It's temporarily and will be replaced by actual service point ID when we decide how to work with service points